data/uking_functions.csv

```diff
@@ -84499,10 +84499,10 @@ Address,Quality,Size,Name
 0x0000007100fd1e80,O,000280,_ZN4ksys15ProductReporter12updateTimersEv
 0x0000007100fd1f98,O,001672,_ZN4ksys15ProductReporter28incrementSceneAndRomWorkTimeEv
 0x0000007100fd2620,U,000076,
-0x0000007100fd266c,U,000620,playReportEmergencyHeapUse
-0x0000007100fd28d8,U,001436,PlayReport::Report::addRomVersions
-0x0000007100fd2e74,U,000688,PlayReport::Report::addPlayTimes
-0x0000007100fd3124,U,000476,PlayReport::Report::addPosition
+0x0000007100fd266c,U,000620,uking::reportEmergencyHeapUse
```

Comment:
Seems like these might be in ksys instead? They're surrounded by ksys functions

Reply:
reportEmergencyHeapUse? It's referenced by DualHeapOrDualFrameHeap, so it probably belongs in ksys.

Reply:
```csv
0x0000007100fd28d8,U,001436,uking::PlayReport::addRomVersions
0x0000007100fd2e74,O,000688,_ZN5uking10PlayReport12addPlayTimesEv
0x0000007100fd3124,O,000476,_ZN5uking10PlayReport11addPositionERKN4sead7Vector2IfEE
```
these functions should probably be in ksys as well, as they are in the ksys PlayReport/PlayReportMgr .cpp file